test/mlir/conversion/onnx_to_torch/squeezeop/squeezeop13_non-negative_axis.onnx.mlir

```diff
@@ -0,0 +1,10 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {}  {
+  func @main_graph(%arg0: tensor<?x1x2x1x2xf32>) -> tensor<?x2x1x2xf32> attributes {input_names = ["input"], output_names = ["output"]} {
+    %0 = "onnx.Constant"() {onnx_node_name = "Constant_0", value = dense<1> : tensor<1xi64>} : () -> tensor<1xi64>
+//CHECK: %[[DIM:.*]] = torch.constant.int 1
+    %1 = "onnx.Squeeze"(%arg0, %0) {onnx_node_name = "Squeeze_1"} : (tensor<?x1x2x1x2xf32>, tensor<1xi64>) -> tensor<?x2x1x2xf32>
+//CHECK: torch.aten.squeeze.dim %arg0, %[[DIM]] : !torch.vtensor<[?,1,2,1,2],f32>, !torch.int -> !torch.vtensor<[?,2,1,2],f32>
```

Comment:
This is also wrong. For `torch.constant.int 0` the result type would be correct. Below you can find a corresponding example I generated from `torch-mlir`.

```
module attributes {torch.debug_module_name = "Model"} {
  func @forward(%arg0: !torch.vtensor<[1,3,128,128],f32>) -> !torch.vtensor<[1,3,128,128],f32> {
    %int1 = torch.constant.int 1
    %0 = torch.aten.squeeze.dim %arg0, %int1 : !torch.vtensor<[1,3,128,128],f32>, !torch.int -> !torch.vtensor<[1,3,128,128],f32>
    return %0 : !torch.vtensor<[1,3,128,128],f32>
  }
}
```